### ISSUE TITLE
missing few files in /vagrant/ansible/... ? 

### ISSUE THREAD

**Issue by maur** [phansible/phansible#226]
I'm using osx as a host. Downloaded generated zip, unpacked, typed "vagrant up" and ended up with the same error.

full log:
```
vagrant up
/opt/vagrant/bin/../embedded/gems/gems/vagrant-1.7.2/lib/vagrant/pre-rubygems.rb:31: warning: Insecure world writable dir /usr/local in PATH, mode 040777
/opt/vagrant/embedded/gems/gems/bundler-1.7.11/lib/bundler/runtime.rb:222: warning: Insecure world writable dir /usr/local in PATH, mode 040777
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'ubuntu/trusty64' is up to date...
==> default: Setting the name of the VM: muz.local
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 => 2222 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
    default:
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default:
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if its present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Configuring and enabling network interfaces...
==> default: Exporting NFS shared folders...
==> default: Preparing to edit /etc/exports. Administrator privileges will be required...
==> default: Mounting NFS shared folders...
==> default: Running provisioner: shell...
    default: Running: /var/folders/0q/56tpbpr54p38mvzl4j9t4wj00000gn/T/vagrant-shell20150619-14362-1q0beoj.sh
==> default: stdin: is not a tty
==> default: Ign http://security.ubuntu.com trusty-security InRelease
==> default: Ign http://archive.ubuntu.com trusty InRelease
==> default: Get:1 http://security.ubuntu.com trusty-security Release.gpg [933 B]
==> default: Ign http://archive.ubuntu.com trusty-updates InRelease
==> default: Hit http://archive.ubuntu.com trusty Release.gpg
==> default: Get:2 http://security.ubuntu.com trusty-security Release [63.5 kB]
==> default: Get:3 http://archive.ubuntu.com trusty-updates Release.gpg [933 B]
==> default: Hit http://archive.ubuntu.com trusty Release
==> default: Get:4 http://archive.ubuntu.com trusty-updates Release [63.5 kB]
==> default: Get:5 http://security.ubuntu.com trusty-security/main Sources [85.8 kB]
==> default: Get:6 http://security.ubuntu.com trusty-security/universe Sources [26.2 kB]
==> default: Get:7 http://archive.ubuntu.com trusty/main Sources [1,064 kB]
==> default: Get:8 http://security.ubuntu.com trusty-security/main amd64 Packages [299 kB]
==> default: Get:9 http://security.ubuntu.com trusty-security/universe amd64 Packages [109 kB]
==> default: Hit http://security.ubuntu.com trusty-security/main Translation-en
==> default: Get:10 http://security.ubuntu.com trusty-security/universe Translation-en [61.4 kB]
==> default: Get:11 http://archive.ubuntu.com trusty/universe Sources [6,399 kB]
==> default: Hit http://archive.ubuntu.com trusty/main amd64 Packages
==> default: Hit http://archive.ubuntu.com trusty/universe amd64 Packages
==> default: Hit http://archive.ubuntu.com trusty/main Translation-en
==> default: Hit http://archive.ubuntu.com trusty/universe Translation-en
==> default: Get:12 http://archive.ubuntu.com trusty-updates/main Sources [210 kB]
==> default: Get:13 http://archive.ubuntu.com trusty-updates/universe Sources [121 kB]
==> default: Get:14 http://archive.ubuntu.com trusty-updates/main amd64 Packages [557 kB]
==> default: Get:15 http://archive.ubuntu.com trusty-updates/universe amd64 Packages [290 kB]
==> default: Get:16 http://archive.ubuntu.com trusty-updates/main Translation-en [269 kB]
==> default: Get:17 http://archive.ubuntu.com trusty-updates/universe Translation-en [151 kB]
==> default: Ign http://archive.ubuntu.com trusty/main Translation-en_US
==> default: Ign http://archive.ubuntu.com trusty/universe Translation-en_US
==> default: Fetched 9,771 kB in 8s (1,182 kB/s)
==> default: Reading package lists...
==> default: Reading package lists...
==> default: Building dependency tree...
==> default: Reading state information...
==> default: software-properties-common is already the newest version.
==> default: 0 upgraded, 0 newly installed, 0 to remove and 12 not upgraded.
==> default: gpg:
==> default: keyring `/tmp/tmpzlsa8n08/secring.gpg' created
==> default: gpg:
==> default: keyring `/tmp/tmpzlsa8n08/pubring.gpg' created
==> default: gpg:
==> default: requesting key 7BB9C367 from hkp server keyserver.ubuntu.com
==> default: gpg:
==> default: /tmp/tmpzlsa8n08/trustdb.gpg: trustdb created
==> default: gpg:
==> default: key 7BB9C367: public key "Launchpad PPA for Ansible, Inc." imported
==> default: gpg:
==> default: Total number processed: 1
==> default: gpg:
==> default:               imported: 1
==> default:   (RSA: 1)
==> default: OK
==> default: Ign http://ppa.launchpad.net trusty InRelease
==> default: Get:1 http://ppa.launchpad.net trusty Release.gpg [836 B]
==> default: Get:2 http://ppa.launchpad.net trusty Release [15.1 kB]
==> default: Get:3 http://ppa.launchpad.net trusty/main amd64 Packages [489 B]
==> default: Ign http://security.ubuntu.com trusty-security InRelease
==> default: Ign http://archive.ubuntu.com trusty InRelease
==> default: Get:4 http://ppa.launchpad.net trusty/main Translation-en [322 B]
==> default: Ign http://archive.ubuntu.com trusty-updates InRelease
==> default: Hit http://security.ubuntu.com trusty-security Release.gpg
==> default: Hit http://archive.ubuntu.com trusty Release.gpg
==> default: Hit http://security.ubuntu.com trusty-security Release
==> default: Hit http://archive.ubuntu.com trusty-updates Release.gpg
==> default: Hit http://security.ubuntu.com trusty-security/main Sources
==> default: Hit http://archive.ubuntu.com trusty Release
==> default: Hit http://security.ubuntu.com trusty-security/universe Sources
==> default: Hit http://archive.ubuntu.com trusty-updates Release
==> default: Hit http://security.ubuntu.com trusty-security/main amd64 Packages
==> default: Hit http://archive.ubuntu.com trusty/main Sources
==> default: Hit http://security.ubuntu.com trusty-security/universe amd64 Packages
==> default: Hit http://archive.ubuntu.com trusty/universe Sources
==> default: Hit http://security.ubuntu.com trusty-security/main Translation-en
==> default: Hit http://archive.ubuntu.com trusty/main amd64 Packages
==> default: Hit http://security.ubuntu.com trusty-security/universe Translation-en
==> default: Hit http://archive.ubuntu.com trusty/universe amd64 Packages
==> default: Hit http://archive.ubuntu.com trusty/main Translation-en
==> default: Hit http://archive.ubuntu.com trusty/universe Translation-en
==> default: Hit http://archive.ubuntu.com trusty-updates/main Sources
==> default: Hit http://archive.ubuntu.com trusty-updates/universe Sources
==> default: Hit http://archive.ubuntu.com trusty-updates/main amd64 Packages
==> default: Hit http://archive.ubuntu.com trusty-updates/universe amd64 Packages
==> default: Hit http://archive.ubuntu.com trusty-updates/main Translation-en
==> default: Hit http://archive.ubuntu.com trusty-updates/universe Translation-en
==> default: Ign http://archive.ubuntu.com trusty/main Translation-en_US
==> default: Ign http://archive.ubuntu.com trusty/universe Translation-en_US
==> default: Fetched 16.7 kB in 4s (3,976 B/s)
==> default: Reading package lists...
==> default: Reading package lists...
==> default: Building dependency tree...
==> default: Reading state information...
==> default: The following extra packages will be installed:
==> default:   python-jinja2 python-markupsafe python-paramiko python-support sshpass
==> default: Suggested packages:
==> default:   python-jinja2-doc
==> default: The following NEW packages will be installed:
==> default:   ansible python-jinja2 python-markupsafe python-paramiko python-support
==> default:   sshpass
==> default: 0 upgraded, 6 newly installed, 0 to remove and 12 not upgraded.
==> default: Need to get 942 kB of archives.
==> default: After this operation, 6,148 kB of additional disk space will be used.
==> default: Get:1 http://ppa.launchpad.net/ansible/ansible/ubuntu/ trusty/main ansible all 1.9.1-1ppa~trusty [624 kB]
==> default: Get:2 http://archive.ubuntu.com/ubuntu/ trusty/universe python-support all 1.0.15 [26.7 kB]
==> default: Get:3 http://archive.ubuntu.com/ubuntu/ trusty/main python-markupsafe amd64 0.18-1build2 [14.3 kB]
==> default: Get:4 http://archive.ubuntu.com/ubuntu/ trusty/main python-jinja2 all 2.7.2-2 [161 kB]
==> default: Get:5 http://archive.ubuntu.com/ubuntu/ trusty/main python-paramiko all 1.10.1-1git1build1 [106 kB]
==> default: Get:6 http://archive.ubuntu.com/ubuntu/ trusty/universe sshpass amd64 1.05-1 [10.5 kB]
==> default: dpkg-preconfigure: unable to re-open stdin: No such file or directory
==> default: Fetched 942 kB in 1s (717 kB/s)
==> default: Selecting previously unselected package python-support.
==> default: (Reading database ... 86712 files and directories currently installed.)
==> default: Preparing to unpack .../python-support_1.0.15_all.deb ...
==> default: Unpacking python-support (1.0.15) ...
==> default: Selecting previously unselected package python-markupsafe.
==> default: Preparing to unpack .../python-markupsafe_0.18-1build2_amd64.deb ...
==> default: Unpacking python-markupsafe (0.18-1build2) ...
==> default: Selecting previously unselected package python-jinja2.
==> default: Preparing to unpack .../python-jinja2_2.7.2-2_all.deb ...
==> default: Unpacking python-jinja2 (2.7.2-2) ...
==> default: Selecting previously unselected package python-paramiko.
==> default: Preparing to unpack .../python-paramiko_1.10.1-1git1build1_all.deb ...
==> default: Unpacking python-paramiko (1.10.1-1git1build1) ...
==> default: Selecting previously unselected package sshpass.
==> default: Preparing to unpack .../sshpass_1.05-1_amd64.deb ...
==> default: Unpacking sshpass (1.05-1) ...
==> default: Selecting previously unselected package ansible.
==> default: Preparing to unpack .../ansible_1.9.1-1ppa~trusty_all.deb ...
==> default: Unpacking ansible (1.9.1-1ppa~trusty) ...
==> default: Processing triggers for man-db (2.6.7.1-1ubuntu1) ...
==> default: Setting up python-support (1.0.15) ...
==> default: Setting up python-markupsafe (0.18-1build2) ...
==> default: Setting up python-jinja2 (2.7.2-2) ...
==> default: Setting up python-paramiko (1.10.1-1git1build1) ...
==> default: Setting up sshpass (1.05-1) ...
==> default: Setting up ansible (1.9.1-1ppa~trusty) ...
==> default: Processing triggers for python-support (1.0.15) ...
==> default: cp:
==> default: cannot stat ‘/vagrant/ansible/inventories/dev’
==> default: : No such file or directory
==> default: cat:
==> default: /vagrant/ansible/files/authorized_keys
==> default: : No such file or directory
==> default: ERROR: the playbook: /vagrant/ansible/playbook.yml could not be found
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

I saw ticket 71, it's not it. I reviewed permissions of directories and files (especially the ones with errors in my log) and everything seems fine:
```
[ ~/work/dev_env/phansible_muz ] ls -ld ansible/inventories/
drwxr-xr-x@ 3 maur  staff  102 19 cze 17:06 ansible/inventories/
[ ~/work/dev_env/phansible_muz ] ls -ld ansible/inventories/dev
-rw-r--r--@ 1 maur  staff  28 19 cze 11:06 ansible/inventories/dev
[ ~work/dev_env/phansible_muz ] ls -ld ansible/playbook.yml
-rw-r--r--@ 1 maur  staff  178 19 cze 11:06 ansible/playbook.yml
```

I already got working some machine with php54, but happy to help if you need me to check anything else here. 
I'm running Yosemite 10.10.3, today I also tested many other similar softs, based on puppet, chef, ansible.. and most of them worked, so my environment should be fine.

**Comment by naxhh** [comment]
Can you check if the `/vagrant` folder has your `~/work/dev_env/phansible_muz` content?
I think is a problem of the mounting point...

I'had some problems on my os X regarding exports (/etc/exports). I don't know what causes the issue but removing the file seems to solve it.
I don't know if that may be your case or not. So if you try remember to backup the system.

**Comment by maur** [comment]
Oh. My bad. I changed "./" dir to absolute path to my project. I was trying phansible right after some different tool which was mounting only project directory. Or mounting config files in the background? Well, nevermind. 
After fixing that - works. I also generated one more time config via the website with older ubuntu, php54 and some modules - works like a charm as well (unless mbstring is selected, but package is missing since it's included in php pkg).
So, closing, sorry for such a silly mistake.
